CMakeLists.txt

```diff
@@ -919,9 +925,9 @@ set(SWIFT_MAIN_INCLUDE_DIR "${SWIFT_SOURCE_DIR}/include")
 set(SWIFT_SHIMS_INCLUDE_DIR "${SWIFT_SOURCE_DIR}/stdlib/public/SwiftShims")
 set(SWIFT_INCLUDE_DIR "${CMAKE_CURRENT_BINARY_DIR}/include")
 
-if (NOT BOOTSTRAPPING_MODE)
-  message(FATAL_ERROR "turning off bootstrapping is not supported anymore")
-endif()
+#if (NOT BOOTSTRAPPING_MODE)
+#  message(FATAL_ERROR "turning off bootstrapping is not supported anymore")
+#endif()
```

Comment:
Please don't disable this check. Instead you can move the first change below this check.

Reply:
Still rebuilding and test, but fixed.